libmozevent/web.py

```diff
@@ -54,6 +60,35 @@ def stop(self):
         self.process.kill()
         logger.info("Web server stopped")
 
+    async def get_version(self, request):
+        """
+        HTTP GET version
+        Following Dockerflow protocol
+        """
+        try:
+            with open(self.version_path, "r") as version_file:
+                version = json.loads(version_file.read())
+        except Exception:
+            return web.HTTPInternalServerError(
+                reason="Could not retrieve the version file"
+            )
+
+        return web.json_response(version)
+
+    async def get_heartbeat(self, request):
+        """
+        HTTP GET heartbeat for backing services
+        Following Dockerflow protocol
+        """
+        return web.HTTPOk()
```

Comment:
I think you are actually returning the header instead of the response.

Just replace that by `return web.Response()` and we suddenly get an empty payload with 200 as header